sdk/core/azure-core/inc/azure/core/base64.hpp

```diff
@@ -21,6 +21,10 @@ namespace Azure { namespace Core {
   class Convert final {
   private:
     // This type currently only contains static methods and hence disallowing instance creation.
+    /**
+     * @brief An instance of `%Convert` class cannot be created.
```

Comment:
What does the `%` symbol do and can you share how this renders in doxygen?

Reply:
https://www.doxygen.nl/manual/autolink.html#linkclass